src/executors/ruby-pg.yml

```diff
@@ -2,13 +2,13 @@ parameters:
   ruby_version:
     description: version of Ruby
     type: string
-    default: latest
+    default: "3.2"
```

Comment:
`cimg/ruby` does not have `latest` tag :sob: . set latest Ruby version on today.

And, I voted https://circleci.canny.io/images/p/re-introduce-latest-tags-for-next-gen-convenience-images . :pray:

Reply:
Also, I reported to https://github.com/CircleCI-Public/cimg-shared/issues/82 .